cinema/serializers.py

```diff
@@ -113,10 +113,10 @@ class Meta:
 class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
This is a code quality issue. It is generally a good practice to provide a docstring for your methods to make your code more understandable. This is especially important for validation methods, as they can contain complex logic. Please consider adding a docstring to the `validate` method to explain what it does, what parameters it takes, what it returns and any exceptions it raises.